roles/ceph-rgw/tasks/rgw_create_pools.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: remove ec profile
+  command: "{{ container_exec_cmd }} ceph --connect-timeout 5 --cluster {{ cluster }} osd erasure-code-profile rm {{ item.value.ec_profile }}"
+  with_dict: "{{ rgw_create_pools }}"
+  delegate_to: "{{ groups[mon_group_name][0] }}"
+  changed_when: false
+  run_once: true
+  when:
+    - item.value.pool_type is defined
+    - item.value.pool_type == 'ec'
+  failed_when: false
+
+- name: set ec profile
+  command: "{{ container_exec_cmd }} ceph --connect-timeout 5 --cluster {{ cluster }} osd erasure-code-profile set {{ item.value.ec_profile }} k={{ item.value.ec_k }} m={{ item.value.ec_m }}"
+  with_dict: "{{ rgw_create_pools }}"
+  delegate_to: "{{ groups[mon_group_name][0] }}"
+  changed_when: false
+  run_once: true
```

Comment:
@dsavineau should all of these be run once or none of these run once? If there are multiple rgw hosts using the same rgws.yml then these commands should only be run once right?

Reply:
right, everything should be run once

I suppose we could add this on the` include_tasks: rgw_create_pools.yml` instead of adding it on every tasks in this file.

WDYT ?

Reply:
I liked it the old way only because having `run_once` for each task makes it clear for anyone making changes or reading that file that each of those tasks is being run once. However, I see that it's redundant and think it's fine.